tests/acceptance/test_multi_threading.py

```diff
@@ -187,7 +187,7 @@ def test___shared_interpreter___run_multiple_acquisitions_with_events___callback
             )
             for i, device in enumerate(multi_threading_test_devices)
         ]
-        samples_per_chan = [1000 for _ in tasks]
+        samples_per_chan = [(100 * (i + 8)) for i in range(len(tasks))]
```

Comment:
`test___shared_interpreter___run_multiple_acquisitions_with_events___callbacks_invoked` doesn't describe what you're testing. Since you are adding new test coverage, please create a new test function that describes what you're testing. As I suggested in chat, "Given shared interpreter & varying sample interval, when I run multiple acquisitions with every N samples events, then requested sample interval passed to callback"

`test___shared_interpreter_varying_sample_interval___run_multiple_acquisitions_with_every_n_samples_events___requested_sample_interval_passed_to_callback`